packages/components/src/text/mixins.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * External dependencies
+ */
+import css from '@emotion/css';
+/**
+ * Internal dependencies
+ */
+import { fontFamily } from './fontFamily';
+
+const fontWeightNormal = `font-weight: normal;`;
+const fontWeightSemibold = `font-weight: 600;`;
+
+const title = `
+  ${ fontWeightNormal }
```

Comment:
⛏ Tiny nit pick
Looks like there may be some indentation inconsistencies (there are other bits in this file as well). If/when we get prettier, it should help with this 😊

Reply:
+1 for `prettier`!